Cargo.toml

```diff
@@ -145,6 +146,9 @@ pretty_assertions = "0.6.1"
 proptest = "0.9"
 tokio01-test = "0.1.1"
 tower-test = "0.1"
+trust-dns-server = "0.16.1"
+trust-dns = "0.16.0"
```

Comment:
just a warning, in `0.18.*` this is being renamed to `trust-dns-client`

Reply:
Oh, and also, is there some reason `0.17` isn't being used here? That would be most compatible with the `0.12` resolver. (This confusion is part of why I'll be switching to versioning all trust-dns crates with a single semver after `0.18`)

Reply:
None that I can remember. So this should be bumped to 0.17.